### PR TITLE
Adds stop_stats.txt for stop-level aggregations

### PR DESCRIPTION
**Pull request by sdrewc**
This adds previously discussed crowding reliability metrics plus a few others.

### CODE REVIEW

**Comment by sdrewc** [comment]
@e-lo This pull is meant to address #6

File: files/stop_stats.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ *  File MUST contain a record for every scheduled stop within a trip and route (i.e. the time when the Muni 14 Local Outbound that left at 8:02 gets to 24th St. and Mission St)
+ *  File MUST be a valid CSV file.
+ *  The first line of each file MUST contain case-sensitive field names.
+ *  Field names MUST NOT contain tabs, carriage returns or new lines.
+ 
+File MUST contain the following required attributes:
+
+Required Attributes	| Description										
+----------			| -------------		
+`trip_id`			| ID that uniquely identifies a vehicle trip
```
Comment (e-lo) [inline files/stop_stats.md:13]: route_id?

File: files/stop_stats.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Required Attributes	| Description										
+----------			| -------------		
+`service_id`		| ID that uniquely identifies a service span in `calendar.txt`.
+
+File MAY contain the following attributes:
+
+Optional Attributes		| Description										
+----------				| -------------		
+`avg_arrival_time_diff`	| Float.  Average difference between observed and scheduled arrival time.
```
Comment (e-lo) [inline files/stop_stats.md:44]: direction_id?

File: files/stop_time_stats.md [hunk 3]
```diff
@@ -30,11 +33,7 @@ Required Attributes	| Description
 
 Required Attributes	| Description										
 ----------			| -------------		
-`trip_id`			| ID that uniquely identifies a vehicle trip
-`stop_id`			| ID that uniquely identifies a stop
 `service_id`		| ID that uniquely identifies a service span in `calendar.txt`.
-`start_time`		| Start time in HH:MM:SS for which route statistics are calculated.
-`end_time`			| End time in HH:MM:SS for which route statistics are calculated.
 
 File MAY contain the following attributes:
```
Comment (e-lo) [inline files/stop_time_stats.md:38]: direction_id? route_id? might not be needed b/c can be looked up from trip_id

File: README.md [hunk 4]
```diff
@@ -3,8 +3,8 @@
 
 A GTFS-based data transit network data standard suitable for dynamic transit modeling.
```
Comment (e-lo) [inline README.md:4]: change descrip?

**Pull request closed without merging**